packages/@snowpack/plugin-webpack/plugin.js

```diff
@@ -159,6 +254,11 @@ module.exports = function plugin(config, args) {
         mode: "production",
         devtool: args.sourceMap ? "source-map" : undefined,
         optimization: {
+          // extract webpack runtime to its own chunk: https://webpack.js.org/concepts/manifest/#runtime
+          runtimeChunk: {
+            name: `webpack-runtime`,
+          },
```

Comment:
Just a question: is this still needed? I remember this being a thing from a while ago, but didn’t know if more recent versions of webpack handled this better (maybe not).

Reply:
I think we still want this. The runtime contains a mapping between chunk IDs and asset names for any dynamic chunks - that's something that could change frequently; you wouldn't want changes to user code to invalidate the caching of vendor chunks.

Webpack 5 does at least make the chunk IDs deterministic by default, but you'd still want to keep the runtime in its own chunk for the same reason.